src/bionics.cpp

```diff
@@ -837,8 +837,8 @@ bool player::uninstall_bionic( std::string const &b_id, int skill_level )
     }
 
     //If you are paying the doctor to do it, shouldn't use your supplies
+    std::vector<item_comp> comps;
     if( skill_level == -1 ) {
-        std::vector<item_comp> comps;
```

Comment:
It's fine at this location, but you need to `#include "requirements.h"`, currently the compiler can not find the type `item_comp` (defined in that header).